core/modules/wbase/SendChannel.h

```diff
@@ -42,16 +42,17 @@ namespace wbase {
 /// accepts only one call to send bytes, unless the sendStream call is used.
 class SendChannel {
 public:
-    typedef util::VoidCallable<void> ReleaseFunc;
-    typedef std::shared_ptr<ReleaseFunc> ReleaseFuncPtr;
-    typedef long long Size;
+    using Ptr = std::shared_ptr<SendChannel>;
+    struct ReleaseFunc {
+        using Ptr = std::shared_ptr<ReleaseFunc>;
+        virtual ~ReleaseFunc() {}
+        virtual void operator()() = 0;
+    };
+    using Size = long long;
```

Comment:
I think `size_t` is more portable, or `ssize_t` if you want signed type. Or maybe `uint64_t` from <cstdint> if you want 64-bit type.